src/material-experimental/mdc-card/card.scss

```diff
@@ -1 +1,163 @@
+// with the previous generation MatCard.
+.mat-mdc-card-content {
+  // Custom elements default to `display: inline`. Reset to 'block'.
+  display: block;
+
+  // Apply default horizontal padding for a content region.
+  padding: 0 $mat-card-default-padding;
+
+  // Only add vertical padding to the main content area if it's not preceeded/followed by another
+  // element within the card.
+  &:first-child {
+    padding-top: $mat-card-default-padding;
+  }
+
+  &:last-child {
+    padding-bottom: $mat-card-default-padding;
+  }
+}
+
+// Title group within a card. MDC does not have a pre-made section for anything title-related.
+// Maintained here for backwards compatibility with the previous generation MatCard.
+.mat-mdc-card-title-group {
+  // This element exists primary to lay out its children (title, subtitle, media). The title
+  // and subtitle go first (projected into a single div), followed by the media.
+  display: flex;
+  justify-content: space-between;
+
+  // Apply default padding for the title-group region. Omit any bottom padding because we assume
+  // this region will be followed by another region that includes top padding.
+  padding: $mat-card-default-padding $mat-card-default-padding 0;
+}
+
+// Avatar image for a card. MDC does not specifically have a card avatar or a "common" avatar.
+// They *do* have a list avatar, but it uses a different size than we do here, which we preserve
+// to reduce breaking changes. Ultimately, the avatar styles just consist of a size and a
+// border-radius.
+.mat-mdc-card-avatar {
+  height: $mat-card-header-size;
+  width: $mat-card-header-size;
+  border-radius: 50%;
+  flex-shrink: 0;
+
+  // Makes `<img>` tags behave like `background-size: cover`. Not supported
+  // in IE, but we're using it as a progressive enhancement.
+  object-fit: cover;
+}
+
+// Specifically sized small image, specific to Angular Material.
+.mat-mdc-card-sm-image {
+  width: 80px;
+  height: 80px;
+}
+
+// Specifically sized medium image, specific to Angular Material.
+.mat-mdc-card-md-image {
+  width: 112px;
+  height: 112px;
+}
+
+// Specifically sized large image, specific to Angular Material.
+.mat-mdc-card-lg-image {
+  width: 152px;
+  height: 152px;
+}
+
+// Specifically sized extra-large image, specific to Angular Material.
+.mat-mdc-card-xl-image {
+  width: 240px;
+  height: 240px;
+}
+
+// Reset the padding for title and subtitle when used within `mat-card-header` or
+// `mat-card-title-group` since the padding is captured by parent container already.
+.mat-mdc-card-header,
+.mat-mdc-card-title-group {
+  .mat-mdc-card-title,
+  .mat-mdc-card-subtitle {
+    padding-top: 0;
+  }
+}
+
+// When both a title and a subtitle are used, eliminate the top padding of whichever comes second
+// because the first already covers the padding.
+.mat-mdc-card-subtitle ~ .mat-mdc-card-title,
+.mat-mdc-card-title ~ .mat-mdc-card-subtitle {
```

Comment:
You could potentially combine it with the declaration above since it's declaring the same styles.

Reply:
Done, though it does make the source less pretty since you have to expand the nesting.